templates/gcp/pro.html

```diff
@@ -0,0 +1,184 @@
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">The right Ubuntu for you</h3>
+            <div class="p-matrix__desc">
+              <p>Canonical brings the three most popular Ubuntu Server distributions to choose from - 14.04 LTS, 16.04 LTS, 18.04 LTS and 20.04 LTS.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">10-year lifetime</h3>
+            <div class="p-matrix__desc">
+              <p>Canonical backs Ubuntu Pro for 10 years<a href="#index-1">*</a>, ensuring security updates are available throughout, with a guaranteed upgrade path.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Optional 24/7 support</h3>
+            <div class="p-matrix__desc">
+              <p>Additional enterprise-grade support available through private offer for Pro.</p>
+            </div>
+          </div>
+        </li>
+      </ul>
+      <p id="index-1"><small>* For 18.04 LTS onwards. The maintenance period for Ubuntu Pro 14.04 LTS and 16.04 LTS is 8 years.</small></p>
+      <a href="https://ubuntu.com/gcp/contact-us" class="p-button--positive">Let's talk Pro</a>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>Broadest open source security coverage keeps CISOs sleeping at night</h2>
+      <p>Ubuntu Pro includes security coverage for the entire collection of software packages shipped with Ubuntu, tracking high and critical CVEs. This makes it ideal for companies that have embraced open source for their new products and need to ensure security for production and mission-critical workloads.</p>
+      <p>One key reason why developers prefer Ubuntu is that it includes a huge set &mdash; in total, over 26,000 packages &mdash; of upstream open source known as 'universe'. Universe includes workloads like NGINX, MongoDB, Redis, Kafka, but also language ecosystems like Python, Java, Ruby and Javascript/Node.js. With Ubuntu Pro's built-in CVE tracking and patching, CIOs and security officers can be confident they can roll these packages out to production.</p>
+    </div>
+    <div class="col-5 col-start-large-8">
+      <blockquote class="p-pull-quote">
+        <p class="p-pull-quote__quote">Ubuntu Pro provides a secure platform for all Ubuntu users on GCP. We are proud to work with Canonical to make this happen, and are continually partnering to improve the experience and security for  all engineering pioneers!</p>
+        <!-- <cite class="p-pull-quote__citation">Christian Reis, VP Public Cloud</cite> -->
```

Comment:
This is a draft quote, should be removed

Reply:
Removed